src/plugins/subsystems/drm/drm.sh

```diff
@@ -208,22 +216,41 @@ function gui_control()
   local isolate_target
   local remote
   local port
+  local set_default='false'
+  local default_command
 
   flag=${flag:-'SILENT'}
 
   if [[ "$operation" == 'ON' ]]; then
     isolate_target='graphical.target'
     vt_console=1
     gui_control_cmd="${configurations[gui_on]}"
-  else
+  elif [[ "$operation" == 'OFF' ]]; then
     isolate_target='multi-user.target'
     vt_console=0
     gui_control_cmd="${configurations[gui_off]}"
+  elif [[ "$operation" == 'ON_AFTER_REBOOT' ]]; then
+    isolate_target='graphical.target'
+    vt_console=1
+    gui_control_cmd="${configurations[gui_on_after_reboot]}"
+    set_default='true'
+  elif [[ "$operation" == 'OFF_AFTER_REBOOT' ]]; then
+    isolate_target='multi-user.target'
+    vt_console=0
+    gui_control_cmd="${configurations[gui_off_after_reboot]}"
+    set_default='true'
```

Comment:
Since these new commands do not affect the interface directly, i.e., it does not enable or disable in the current session, it would be nice to add a message to let the user know that their change will only take effect after reboot and if they want it in the current session they can use the `--gui-on/off` option.